core/platform/models_test.py

```diff
@@ -276,10 +276,11 @@ def test_import_cache_services(self):
 
     def test_import_taskqueue_services(self):
         """Tests import taskqueue services function."""
-        from core.platform.taskqueue import gae_taskqueue_services
-        self.assertEqual(
-            self.registry_instance.import_taskqueue_services(),
-            gae_taskqueue_services)
+        with self.swap(constants, 'DEV_MODE', False):
```

Comment:
Technically you should also swap utils.is_appengine_development_environment(), right?

And also test the other cases?